evals/registry/solvers/whisper.yaml

```diff
@@ -0,0 +1,36 @@
+generation/whisper/gpt-4-turbo:
+  #solver: OpenAISolver
+  class: evals.solvers.providers.openai.whisper_solver:WhisperSolver
+  args:
+    solver:
+      class: evals.solvers.providers.openai.openai_solver:OpenAISolver
+      args:
+        completion_fn_options:
+          model: gpt-4-turbo
```

Comment:
nit: I suggest `gpt-4o-mini` instead (faster and not worse :?)

Reply:
agreed, faster will be better. But not currently supported in `evals`, will revisit once my other PRs land